src/main/java/hw_jdi/entities/User.java

```diff
@@ -0,0 +1,15 @@
+package hw_jdi.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class User {
+
+    public static User ROMAN = new User("Roman", "Jdi1234", "ROMAN IOVLEV");
```

Comment:
It will be better to read this entity from config file, since you have that one.